### DISCUSSION TITLE
Calendar not working with right sidebar

### DISCUSSION THREAD

**Discussion by cuongseo** [fomantic/Fomantic-UI#1047] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
# Bug Report

## Steps to reproduce
1.Calendar not working with right sidebar

## Expected result
Show calendar in input field
## Actual result
Calendar not show
## Screenshot (when possible)
![]()

## Version
x.x.x

**Comment by exoego** [comment]
> 1.Calendar not working with right sidebar

I think this is not enough to reproduce and understand this issue .
Please consider forking https://jsfiddle.net/31d6y7mn and create a minimum reproducible demo.

**Comment by lubber-de** [comment, marked as answer]
Basically a calendar works in a right sidebar
https://jsfiddle.net/vwqmytpn/

But, what the issuer probably claims about is the fact that the sidebar has a `overflow-y` set which cuts the calendar, because it's larger than the default sidebar width (we have similiar problems when using calendar or popups in scrollable dropdowns or modals)
![image](https://user-images.githubusercontent.com/18379884/65963929-127fcb80-e45c-11e9-80fd-69d0dacc1513.png)

There are 2 solutions here

### 1) use a `wide sidebar`
https://jsfiddle.net/vwqmytpn/1/
![image](https://user-images.githubusercontent.com/18379884/65964061-5e327500-e45c-11e9-8d65-2874a50a6a2c.png)

### 2) override the settings for `overflow` and `z-index` and set the popup position accordingly
https://jsfiddle.net/vwqmytpn/2/
![image](https://user-images.githubusercontent.com/18379884/65965634-3bee2680-e45f-11e9-934c-863380317376.png)

**Comment by lubber-de** [comment]
Closing because assumed question was answered. Happy to reopen, if your issue was different and you provide a reproducable jsfiddle.
